tensorflow/core/kernels/bucketize_op.h

```diff
@@ -32,7 +32,16 @@ struct BucketizeFunctor {
   static Status Compute(OpKernelContext* context,
                         const typename TTypes<T, 1>::ConstTensor& input,
                         const std::vector<float>& boundaries_vector,
-                        typename TTypes<int32, 1>::Tensor& output);
+                        typename TTypes<int32, 1>::Tensor& output) {
+    const int N = input.size();
```

Comment:
Function needs to be rewritten not to use loops and access input and output with .device()
Put back the CPU template specialization in the cc file

Reply:
Fixed.